_data/simple-icons.json

```diff
@@ -8219,6 +8219,11 @@
             "hex": "CD1414",
             "source": "https://www.penny.de/"
         },
+        {
+            "title": "penpot",
```

Comment:
```suggestion
            "title": "Penpot",
```

Typo.

Reply:
Here, I used the logo text as the title.

Reply:
See here to get to know how they spell it: https://penpot.app/about.html

Reply:
> They should keep the same case for the logo.

No. There are different things. They can spell it to anything in the logo they want.